kolibri/plugins/coach/assets/src/views/reports/ReportsLessonReportPage.vue

```diff
@@ -30,15 +30,15 @@
                 :text="tableRow.title"
                 :to="classRoute(
                   'ReportsLessonExerciseLearnerListPage',
-                  { exerciseId: tableRow.content_id }
+                  { exerciseId: tableRow.node_id }
```

Comment:
this doesn't seem right.

I've been treating all URL params as `content_id`s, not `node_id`s

Reply:
Hrm, I made this edit early on to get stuff working. I think the major issue was that the node id is needed to pass into the exercise detail view handlers to get the specific exercise content, including the assessmentmetadata. Probable that I can find the node_id in another way.